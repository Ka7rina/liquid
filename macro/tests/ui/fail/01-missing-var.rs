use liquid_macro::seq;

seq!(N of 1..10 {
    // nothing
});

fn main() {}
