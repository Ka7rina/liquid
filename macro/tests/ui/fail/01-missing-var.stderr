error: expected `in`
 --> $DIR/01-missing-var.rs:3:8
  |
3 | seq!(N of 1..10 {
  |        ^^
